### PR TITLE
Update FAQ: TAN-Hotline Shutdown

### PR DESCRIPTION
**Pull request by larswmh**
This PR modifies FAQ entries containing information about the TAN-Hotline, in accordance with its scheduled discontinuation after January 31st, 2023. 

---
Internal Tracking ID: [EXPOSUREAPP-14496](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14496)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -371,8 +371,8 @@
                                 "anchor": "warn_for_others",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated January 18, 2023</b>",
-                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 onwards and cannot be used with the previous versions 2.9 to 2.28 after January 31, 2023, as the verification hotline (TAN hotline) will be discontinued on this date.",
+                                    "<b>Updated February 1, 2023</b>",
+                                    "This feature of the Corona-Warn-App is no longer available since version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after January 31, 2023, as the verification hotline (TAN hotline) was be discontinued on this date.",
```
Comment (MikeMcC399) [inline src/data/faq.json:375]: "was be" => "was"

File: src/data/faq.json [hunk 2]
```diff
@@ -1418,8 +1419,8 @@
                                 "anchor": "check_in_luca_warn_for_others",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated January 18, 2023</b>",
-                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 onwards and cannot be used with the previous versions 2.9 to 2.28 after January 31, 2023, as the verification hotline (TAN hotline) will be discontinued on this date.",
+                                    "<b>Updated February 1, 2023</b>",
+                                    "This feature of the Corona-Warn-App is no longer available since version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after January 31, 2023, as the verification hotline (TAN hotline) will be discontinued on this date.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1423]: "will be" => "was"

File: src/data/faq.json [hunk 3]
```diff
@@ -1730,9 +1731,9 @@
...
+                                    "The operation of the TAN hotline has been discontinued as of January 31, 2023. Previously, it helped you enter the TAN if you had a positive PCR test result and if you did not receive a TAN or a document with a PCR test QR code. You could call them from Monday to Sunday from 7 a.m. to 8 p.m. local time in Germany. The call was free of charge within Germany. Please note that since version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that was not submitted to the Corona-Warn-App.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1736]: Not really helpful to leave the operating hours and other details here.

File: src/data/faq.json [hunk 4]
```diff
@@ -1748,17 +1749,16 @@
...
                                     "<b>TAN Hotline</b>",
-                                    "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time:",
-                                    "<b>National:</b> 0800 7540002",
-                                    "<b>International:</b> +49 30 498 75402",
-                                    "Please note that as of version 3.0 of the Corona-Warn-App, a teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of January 31, 2023.",
+                                    "The operation of the TAN hotline has been discontinued as of January 31, 2023. Previously, it helped you enter the TAN if you had a positive PCR test result and if you did not receive a TAN or a document with a PCR test QR code. You were able to call it from Monday to Sunday from 7 a.m. to 8 p.m. local time in Germany:",
```
Comment (MikeMcC399) [inline src/data/faq.json:1759]: Same here regarding operating hours.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Changes needed in English version.

Also I made the comment that the operating hours etc. are no longer relevant and could be removed.

**Comment by MikeMcC399** [comment]
I called the TAN hotline number and the recording is already live that it has been discontinued.

File: src/data/faq.json (hunk 4 above)
Comment (larswmh) [reply src/data/faq.json]: Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!

File: src/data/faq.json (hunk 3 above)
Comment (larswmh) [reply src/data/faq.json]: Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!

File: src/data/faq.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq.json]: Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!

File: src/data/faq.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq.json]: Updated in https://github.com/corona-warn-app/cwa-website/pull/3357/commits/b006857fcb5fdb081ce62f9f90250306044fa7f1, thanks for the suggestion!

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for assisting here !

**Pull request merged**